README.md

```diff
@@ -12,7 +12,7 @@ This repository is for providing feedback and documentation on the Pylance langu
 
 # Features
 
-![ screencast ](screencap.gif)
+<img src=https://raw.githubusercontent.com/microsoft/pylance-release/images/all-features.gif>
```

Comment:
I'd leave this as relative, then in our VSIX we can make it absolute to try and fix the bug.

Reply:
Done deal